docs/2_4_common_schema.adoc

```diff
@@ -659,129 +664,76 @@ _This information can, for example, be used to increase numerical stability and
-_For maximum performance on production or real-time systems, these checks might not be performed._ +
-_The approach in FMI is therefore that min/max definitions are an information from the FMU to the environment defining the region in which the FMU is designed to operate._
-_In any case, it is expected that the FMU handles variables appropriately where the region definition is critical._
-_For example, dividing by an <<input>> (so the <<input>> should not be in a small range of zero) or taking the square root of an <<input>> (so the <<input>> should not be negative) may either result in <<fmi3Error>>, or the FMU is able to handle this situation in other ways._
-
-_If the FMU is generated so that min/max shall be checked whenever meaningful (for example, for debug purposes), then the following strategy should be used:_
-
-_If `fmi3Set{VariableType}` is called violating the min/max attribute settings of the corresponding variable, the following actions are performed:_
-
-- _On a <<fixed>> or <<tunable>> <<parameter>> <<fmi3Discard,`fmi3Status == fmi3Discard`>> is returned._
-- _On an <<input>>, the FMU decides what to return (If no computation is possible, it could return <<fmi3Discard,`fmi3Status == fmi3Discard`>>, in other situations it may return <<fmi3Warning>> or <<fmi3Error>>, or <<fmi3OK>>, if it is uncritical)._
-
-_If an FMU defines min/max values for integer types and `<Enumeration>` variables (<<local>> and <<output>> variables), then the expected behavior of the FMU is that `fmi3Get{VariableType}` functions return values in the defined range._
-
-_If an FMU defines min/max values for numeric types, then the expected behavior of the FMU is that `fmi3Get{VariableType}` returns values at the solution (accepted steps of the integrators) in the defined range with a certain uncertainty related to the tolerances of the numerical algorithms.]_
-
-===== Clock Type Definition [[clock-type-definition]]
-
-Clocks are integrated in the element `<fmiModelDescription><ModelVariables>` as a variable element with the base type `fmi3Clock`.
-The variable sub type `fmi3Clock` provides additional attributes for defining <<clock,`clocks`>>.
-
-.Clock attribute details.
-[[table-clock-attributes]]
-[cols="1,5",options="header"]
-|====
-|Attribute
-|Description
-
-|`clockType`
-
-a|
-[[clockType,`clockType`]]
-The type of <<clock,`clocks`>> is defined based on the mandatory attribute `clockType` with the following two values:
-
-[[synchronousTime,`synchronousTime`]]
-`= synchronousTime` is used, if the properties of the <<clock>> adhere to synchronous clock theory.
 
-[[communicationPoint,`communicationPoint`]]
-`= communicationPoint` <<clock,`clocks`>> define sampling points (i.e. communication points) for the variables of model partitions.
-
-It is not allowed to include <<clock,`clocks`>> of different `clockType` in one FMU.
-`clockType` is a required attribute.
+2+|Attributes for `ClockType`
 
 |`triggeredBy`
```

Comment:
Isn't this information that belongs to the model structure?

Reply:
In principle yes, maybe. But this is the dependency of an inputClock on the tick of an outputClock - the direction is opposite to "normal" dependencies - we discussed this with Irina and Matthias at length, we should leave as is.